### ISSUE TITLE
Adding and blocking users from their user profiles

### ISSUE THREAD

**Issue by Pjb518** [UnicornHeartClub/Issues#42]
In addition to searching for friends using the "Friend Finder", or potentially blocking them #41 using the same mechanism, users should be able to add or remove a user from their friend list, or block the user from that user's user profile. Right now, the only thing that users can do from when on another user's profile page is message them.

**Comment by tlackemann** [comment]
Absolutely. In fact a whole social feature is planned that already accounts for this. I'm going to close for now as this ties into a much larger release coming out Q1 2018.
